src/clj/game/core/hosting.clj

```diff
@@ -71,7 +71,7 @@
 
        ;; events should be registered for: runner cards that are installed; corp cards that are Operations, or are installed and rezzed
        (when (or (is-type? target "Operation")
-                 (is-type? target "Event")
+                 (and (is-type? target "Event") (false? facedown))
```

Comment:
I can't see the game introducing "facedown" as an identical status for corp side; I imagine they'd name it something else. I think we're safe with assuming that keyword is for runner cards only.